ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -122,6 +122,7 @@ message DamlSubmitterInfo {
   string command_id = 2;
   string application_id = 3;
   google.protobuf.Timestamp maximum_record_time = 4;
+  google.protobuf.Timestamp deduplicate_until = 5;
```

Comment:
If the change is backwards incompatible and we cannot pick a default value for deduplicate_until, we should in principle bump kvutils version to make sure we fail in predictable manner (rather than with obscure decoding error). Have a look at kvutils/Version.scala and possibly bump the version. Also add a mention in that file about what changed and what the semantics are going forward.